### ISSUE TITLE
regarding Instruction adherence 

### ISSUE THREAD

**Issue by meaem** [qishenghu/CodeInstruct#5]
As mentioned in the paper section  **(H) "Alignment of GPT-4 Evaluation and Human Evaluation**", 
I can not figure out in your code how did you performed the instruction adherence of the generated code. 
Was this done manually or automatically?
If automatically,  Could you guide me in which file I can find this code.
Thanks

**Comment by meaem** [comment]
I mean ... to check how the generated code is adhered with the instruction given to the Model .

**Comment by likaixin2000** [comment]
Hi @meaem, thank you for your interest in our work. 

In the paper, we did *both* manual and automatic (GPT-4) evaluation to check the instruction adherence, and calculated the Cohen’s Kappa value between these two types of evaluation. This is done to validate the idea of using GPT as a convenient evaluator. 

The code used for GPT evaluation in Appendix H is not included in this repo. We provided the prompt for GPT evaluation in Appendix C: 
```
Given a code editing instruction, please determine if the output is an acceptable edited code response to the instruction
and input.
```

Hope this answers your question :)
